src/k8s/pkg/utils/cidr.go

```diff
@@ -75,6 +78,19 @@ func GetKubernetesServiceIPsFromServiceCIDRs(serviceCIDR string) ([]net.IP, erro
 
 // ParseAddressString parses an address string and returns a canonical network address.
 func ParseAddressString(address string, port int64) (string, error) {
+	re := regexp.MustCompile(`^\[?([a-z0-9:.]+\/[0-9]+)\]?`)
```

Comment:
Should we add some example matching strings/comments to make it easier to understand this regex?